packages/react-core/package.json

```diff
@@ -29,7 +29,8 @@
   },
   "dependencies": {
     "@aws-amplify/ui": "3.14.0",
-    "@xstate/react": "3.0.0"
+    "@xstate/react": "3.0.1",
```

Comment:
This package needed to be bumped so that the minimum version of `xstate` is in-line with what we install.